replaced static with dynamically generated copyright year

shailysangwan opened the pull request:
https://openedx.atlassian.net/browse/DOC-3212
with reference to the [outreachy page](https://openedx.atlassian.net/wiki/display/COMM/Outreachy+Round+13)

edx-webhook commented:
Thanks for the pull request, @shailysangwan! I've created [OSPR-1435](https://openedx.atlassian.net/browse/OSPR-1435) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).



We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](http://open.edx.org/sites/default/files/wysiwyg/individual-contributor-agreement.pdf)     or indicated your institutional affiliation.     If you like, you can add yourself to the [AUTHORS](https://github.com/edx/edx-platform/blob/master/AUTHORS)      file for this repo, though that isn't required.     Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

shaunagm commented:
@lamagnifica

(Also, @shailysangwan, your agreement has been processed.)

shaunagm commented:
jenkins run all

lamagnifica commented on docs/en_us/enrollment_api/source/conf.py:
```diff
@@ -95,5 +96,5 @@
-project = u'edX Enrollment API Version 1'
-copyright = u'2015, edX'
+project = 'edX Enrollment API Version 1'
+copyright = '{year}, edX'.format(year=datetime.datetime.now().year)
```
please add "Inc." -- > '{year}, edX Inc.'

lamagnifica commented on docs/shared/conf.py:
```diff
@@ -63,8 +64,8 @@ def add_base(paths):
-project = u'edX'
-copyright = u'2013, EdX Doc Team'
+project = 'edX'
+copyright = '{year}, EdX Doc Team'.format(year=datetime.datetime.now().year)
```
eeek! Ok, these should all be roughly the same, with the year and "edX Inc." as the copyright holder.

Somehow, even with all three of these files including three different copyright holders, the published guide is getting and applying a single footer and it seems to be the one in docs/en_us/platform_api/source/conf.py (see http://edx.readthedocs.io/projects/edx-platform-api/en/latest/index.html).
That one is the best of the bunch!

shailysangwan replied:
Is there a script like run_tests.sh from edx_documentation, that I could use here to generate the build directory, or should I start learning sphinx from scratch?

I'd like to see which conf.py files affect the output and change them accordingly.

lamagnifica commented:
In this repo  you can use make commands. Change to the top document directory (like enrollment_api) and run `make clean` then `make html`.

shailysangwan commented:
I ran that on enrollment_api, it worked fine there.
But in platform_api, I got the following errors:

`➜  platform_api git:(master) make clean
rm -rf build/*
➜  platform_api git:(master) make html
sphinx-build -b html  -d build/doctrees -c source  -W source build/html
Running Sphinx v1.3.1
making output directory...
loading pickled environment... not yet created
loading intersphinx inventory from http://docs.python.org/objects.inv...
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 20 source files that are out of date
updating environment: 20 added, 0 changed, 0 removed
reading sources... [100%] user/preferences                                                                                                             

Warning, treated as error:
edx-platform/docs/en_us/platform_api/source/enrollment/enrollment.rst:19: WARNING: autodoc: failed to import class u'EnrollmentView' from module u'enrollment.views'; the following exception was raised:
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 385, in import_object
    __import__(self.modname)
  File "edx-platform/common/djangoapps/enrollment/views.py", line 11, in <module>
    from course_modes.models import CourseMode
  File "edx-platform/common/djangoapps/course_modes/models.py", line 8, in <module>
    from config_models.models import ConfigurationModel
  File "edx-platform/common/djangoapps/config_models/models.py", line 5, in <module>
    from django.contrib.auth.models import User
  File "/usr/lib64/python2.7/site-packages/django/contrib/auth/models.py", line 6, in <module>
    from django.contrib.contenttypes.models import ContentType
  File "/usr/lib64/python2.7/site-packages/django/contrib/contenttypes/models.py", line 138, in <module>
    class ContentType(models.Model):
  File "/usr/lib64/python2.7/site-packages/django/db/models/base.py", line 113, in __new__
    "INSTALLED_APPS." % (module, name)
RuntimeError: Model class django.contrib.contenttypes.models.ContentType doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.

Makefile:56: recipe for target 'html' failed
make: *** [html] Error 1
➜  platform_api git:(master) `

shaunagm commented:
jenkins run all

shaunagm commented:
@shailysangwan That's an unusual error given the changes you've made, and it suggests to me an issue with setting up the development environment rather than a problem with the PR.  I've triggered the automated tests - we'll see if they produce the same error and, if so, we can try and get to the bottom of it.

shailysangwan commented:
It could have something to do with my build environment.
I got another error before this that had something to do with django not being installed, so I installed that.

shailysangwan commented:
Yes, it was a fault on my end. I had not installed all dependencies given in requirements.txt

shailysangwan commented:
Okay, I've changed the copyright message in shared/conf.py and removed it from all other conf.py files.
I have also verified that it appears correctly in the built files.

shaunagm commented:
This looks good to me.  It's not clear to me that we want to keep all of the statements if only one is actually being used - do we want to delete the statements from the other two files?  I'll move this back for the docs team to review and they'll let us know what they think.

lamagnifica commented:
👍 😄

shaunagm commented:
Is this ready to merge, @lamagnifica?  If so, feel free to push the button!

lamagnifica commented:
@shailysangwan please squash your commits and I will merge. Thanks.
